### PR TITLE
Add new subnets, routes and nat-gateway

### PR DESCRIPTION
**Pull request by tianomagdaong**

### CODE REVIEW

File: charts/child-account-composition/templates/composition.yaml [hunk 1]
```diff
@@ -1284,62 +1513,541 @@ spec:
...
+          subnetIdRef:
+            name: #patchme
+    patches:
+    - type: FromCompositeFieldPath
+      fromFieldPath: spec.id
+      toFieldPath: metadata.name
+      transforms:
+      - type: string
+        string:
+          fmt: 'private-%s-1-1c'
```
Comment (jfmrm) [inline charts/child-account-composition/templates/composition.yaml:1699]:
```suggestion
          fmt: '%s-nat-gateway-1'
```

File: charts/child-account-composition/templates/composition.yaml [hunk 2]
```diff
@@ -1284,62 +1513,541 @@ spec:
...
+          fmt: '%s-public-1'
+    - type: FromCompositeFieldPath
+      fromFieldPath: spec.id
+      toFieldPath: spec.forProvider.gatewayIdRef.name
+    - type: FromCompositeFieldPath
+      fromFieldPath: spec.parameters.region
+      toFieldPath: spec.forProvider.region
+    - type: FromCompositeFieldPath
+      fromFieldPath: status.id
+      toFieldPath: spec.providerConfigRef.name
```
Comment (jfmrm) [inline charts/child-account-composition/templates/composition.yaml:1958]: no need for new routes

File: charts/child-account-composition/templates/composition.yaml [hunk 3]
```diff
@@ -1284,62 +1513,541 @@ spec:
...
+      transforms:
+      - type: string
+        string:
+          fmt: '%s-private-1'
+    - type: FromCompositeFieldPath
+      fromFieldPath: spec.id
+      toFieldPath: spec.forProvider.natGatewayIdRef.name
+      transforms:
+      - type: string
+        string:
```
Comment (jfmrm) [inline charts/child-account-composition/templates/composition.yaml:2033]: no need for new routes

**Review by jfmrm — CHANGES_REQUESTED** [review]

File: charts/child-account-composition/templates/composition.yaml [hunk 4]
```diff
@@ -1284,62 +1513,541 @@ spec:
...
+      fromFieldPath: spec.id
+      toFieldPath: spec.forProvider.vpcIdRef.name
+    - type: FromCompositeFieldPath
+      fromFieldPath: spec.parameters.region
+      toFieldPath: spec.forProvider.region
+    - type: FromCompositeFieldPath
+      fromFieldPath: status.id
+      toFieldPath: spec.providerConfigRef.name
+    - type: ToCompositeFieldPath
+      fromFieldPath: status.atProvider.id
```
[the remark this answers is not in the document]
Comment (tianomagdaong) [reply charts/child-account-composition/templates/composition.yaml:1852]: added new routetable-private-1 for the new nat-gateway. removed new public route-table

File: charts/child-account-composition/templates/composition.yaml (hunk 1 above)
Comment (tianomagdaong) [reply charts/child-account-composition/templates/composition.yaml]: done.

File: charts/child-account-composition/templates/composition.yaml (hunk 2 above)
Comment (tianomagdaong) [reply charts/child-account-composition/templates/composition.yaml]: Done.

File: charts/child-account-composition/templates/composition.yaml (hunk 3 above)
Comment (tianomagdaong) [reply charts/child-account-composition/templates/composition.yaml:1964]: Retain new private-route for the new nat-gateway.

File: charts/child-account-composition/templates/composition.yaml [hunk 5]
```diff
@@ -1072,6 +1072,51 @@ spec:
...
+        policy:
+          resolve: "Always"
+    patches:
+    - type: FromCompositeFieldPath
+      fromFieldPath: spec.id
+      toFieldPath: metadata.name
+      transforms:
+      - type: string
+        string:
+          fmt: 'public-%s-1a'
```
[the remark this answers is not in the document]
Comment (tianomagdaong) [reply charts/child-account-composition/templates/composition.yaml]: done.

**Review by jfmrm — APPROVED** [review]

**Pull request merged**